docs/index.rst

```diff
@@ -898,17 +904,26 @@ Dialogues
        "url": "/projects/23/dialogues/21",
        "is_archived": false,
        "is_published": false,
-       "has_changes": false
+       "has_changes": false,
+       "can_view": true,
+       "can_edit": true
      }]
 
   :query boolean is_archived:
-    If ``false``, only return unarchived dialogues. If ``true``, only return archived dialogues. If omitted, both archived and unarchived dialogues are retrieved.
+    Filter on whether this dialogue has been archived.
 
   :query boolean is_published:
-    If ``false``, only return unpublished dialogues. If ``true``, only return published dialogues. If omitted, both published and unpublished dialogues are retrieved.
+    Filter on whether this dialogue has been published.
 
   :query boolean has_changes:
-    If ``false``, only return dialogues without unpublished changes. If ``true``, only return dialogues with unpublished changes. If omitted, dialogues with published and unpublished changes are retrieved.
+    Filter on whether this dialogue has unpublished changes or not.
+
+  :query boolean can_view:
+    Filter on whether the user can view this dialogue or not.
```

Comment:
At the moment, the plan is to allow users with access to a project to be able to view all dialogues that exist in a project, but not able to view how a dialogue looks if they don't have read access for that dialogue.

I think users might have an incomplete picture of what is contained in a project if we don't allow them to see which dialogues are in the project (they don't know whether a dialogue doesn't exist, is actually in a different project, or whether they just don't have access). Seems a bit better to let them know which dialogues exist in the project, but only allow them to view it if they have the right permissions.